dist/js/axs_testing.js

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
```

Comment:
Do we want to be generating a new `axs_testing.js` in this PR? Seems like the PR should be limited to changes to the release process itself.